test.php

```diff
@@ -64,9 +64,10 @@
 if (!$check) $ok++;
 print "Check wrong: '" . $check . "' (should be '0' or '')\n";
 
-if ($ok === 6)
+if ($ok === 6) {
 	print "All tests have PASSED\n";
-else
+	exit(0);
+} else {
 	print "Some tests have FAILED\n";
-
-?>
+	exit(1);
+}
```

Comment:
Given that we have `<?php` at the beginning, we should probably keep the `?>` here - or drop both.

Reply:
Ah, no.

Okay, so the `<?php` at the beginning of a file is _required_ for the code to be interpreted by the PHP engine.

The `?>` at the end of a file is _optional_ and nowadays, the consensus is to leave it out as it will prevent typically issues with "headers already send" due to the whitespace at the end of the file being interpreted as HTML whitespace and send to the server.

Reply:
Oh, I didn't know of such rationale and consensus. OK, but maybe it's better to make this change to both scripts in here at once, and in a separate commit.

Reply:
No worries. As you mentioned that you are not really into PHP, I'm happy to explain things along the way.

I'll move the removal of the `?>` close tag to a separate PR. Would you be okay with me doing some additional coding style/current best practices clean up in the same PR ? There would be no functional changes, just coding practices changes.

Reply:
Thanks. Yes, OK to have "some additional coding style/current best practices clean up in the same PR", but separate commit(s) from the removal of `?>`.

Reply:
> ... but separate commit(s) from the removal of `?>`.

Ha! A like-minded person, that's how I mostly work anyway. ;-)